docs/api/rest_api/rest_api_authentication.md

```diff
@@ -18,6 +18,11 @@ For other security related subjects, see:
 - [Cross-origin requests](rest_responses.md#cross-origin)
 - [`access_control`]([[= symfony_doc =]]/security/access_control.html)
 
+!!! caution "SiteAccess login"
+
+    The anonymous user is used to access the ressource even if authentification with the REST server has been established.
+    Anonymous user must have `user/login` policy on the SiteAccess matched by the REST domain or passed through [`X-Siteaccess` header](rest_requests.md#siteaccess).
+
```

Comment:
```suggestion
!!! caution "SiteAccess login"

    Even if you authenticate with the REST server, an anonymous user is used to access the resources.
    Therefore, anonymous users must have `user/login` permission on the SiteAccess that matches the REST domain or is passed through the [`X-Siteaccess` header](rest_requests.md#siteaccess).

```

Reply:
@dabrt  I changed this a bit since your review.

I might need a dev review for the first sentence. I'm not sure about why exactly this is needed.